lib/devices.ts

```diff
@@ -316,7 +316,7 @@ export class BalenaFin extends DeviceInteractor {
 						clearTimeout(timeout);
 						console.log(`DEBUG: Timed out!`);
 						reject();
-					}, 1000 * 60 * 5);
+					}, 1000 * 60);
```

Comment:
A bit scary with how short it is, but lets try it I guess

Reply:
it should just be for mounting the fin but I dunno lets see